packages/next-app/pages/[kanbanId]/index.tsx

```diff
@@ -168,3 +176,29 @@ function NewTask({ task, isPM, kanban, address }) {
     </div>
   );
 }
+
+function ProgressTask({ task, kanban, }) {
+  const { id, title, detail, requests, funds,raider } = task;
+  const [loading, setLoading] = useState(false);
+  
+  const onSubmitTask = async () => {
+    const kb = kanban as Kanban;
+    const tasklog = await kb.taskLog(id); 
+    tasklog.reviewed;
+  };
+
+  return (
+    <div className="h-40 w-full p-3 m-2 border text-gray-400">
+      <div className="text-2xl text-black">{title}</div>
+      <div className="h-6">{detail}</div>
+      <div className="h-6">{raider.id}</div>
+      <div className="text-sm mt-2">{ethers.utils.formatEther(funds)} ETH</div>
+      <button
```

Comment:
This button should only be visible for the raider assigned to the task. Do a comparison like: const tmp = ethers.utils.getAddress(raider.id); tmp.localeCompare(address) === 0; This will tell you if the logged in user is the assigned raider.